routes/auth.js

```diff
@@ -0,0 +1,105 @@
+var express = require("express");
+var router = express.Router();
+var passport = require("passport");
+var User = require("../models/user");
+var Score = require("../models/score");
+
+//------------------- AUTH ROUTES------------------
+
+// show register form
+router.get("/register", function(req, res) {
+  res.render("register.ejs");
+});
+
+//handle sign up logic
+router.post("/register", function(req, res) {
```

Comment:
i suggest you have 3 layers:
http layer -> handles routes, request / response, http errors etc...
logic layer --> this is your actual ear training logic
infra layer --> data access files, emails, and other infra stuff

when you write test you'll write most of them on the logic layer